relay/sources/relayd/tools/config/main.conf

```diff
@@ -56,3 +56,5 @@ verify_certificates = true
 command = "/opt/rudder/bin/rudder"
 use_sudo = true
 
+[shared_files_path]
```

Comment:
The section shoule probably be named `shared_files` as it may contains other information